### ISSUE TITLE
Update settings in OOD 3.1.x to allow custom logo control of height and width

### ISSUE THREAD

**Issue by rlara3** [OSC/ondemand#3630]
On OOD 3.0, the setting for the size of the home screen logo controlled the height, with the width setting as proportionate:

```
OOD 3.0
<img src="/public/sos_elsa.png?timestamp=1708715652" alt="logo" class="py-2">
```

Example:

![image](https://github.com/OSC/ondemand/assets/45435166/cc86bfed-a124-42ef-9247-eb196060af85)

With OOD 3.1, the width is set at `100`, creating issues with the image size. Large and small images are scaled up to the width of the page, creating formatting issues:

```
OOD 3.1.1
<img src="/public/sos_elsa.png?timestamp=1708717039" alt="logo" class="py-2 w-100">
```

Example:

![image](https://github.com/OSC/ondemand/assets/45435166/d88a262b-3cd7-466d-88c0-afe4e2df6984)

The setting is in the file `nginx-stage.yml`.

**Comment by johrstrom** [comment]
Hi, thanks for this ticket. This has been patched in the next release, but i'll put this in the 3.1 milestone so that we can pick it up and patch it in some 3.1.x release.

**Comment by johrstrom** [comment]
As a workaround until it gets released, you can edit this file located `/var/www/ood/apps/sys/dashboard/app/helpers/dashboard_helper.rb` 

https://github.com/OSC/ondemand/blob/b4e699860a0e358717d6066bef44f64288f566b6/apps/dashboard/app/helpers/dashboard_helper.rb#L16

And change this to remove the `w-100` class.

```diff
- image_tag("OpenOnDemand_stack_RGB.svg", alt: "logo", height: "85", class: 'py-2 w-100') 
+ image_tag("OpenOnDemand_stack_RGB.svg", alt: "logo", height: "85", class: 'py-2') 
```

**Comment by johrstrom** [comment]
OK - looks like you need 2 edits in the same function. You can check this backport for the actual diff you need: #3632

**Comment by rlara3** [comment]
Found the second edit, all is good now.

Ruben Lara
pace.gatech.edu

From: Jeff Ohrstrom ***@***.***>
Date: Monday, June 24, 2024 at 1:49 PM
To: OSC/ondemand ***@***.***>
Cc: Lara, Ruben ***@***.***>, Author ***@***.***>
Subject: Re: [OSC/ondemand] Update settings in OOD 3.1.x to allow custom logo control of height and width (Issue #3630)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

OK - looks like you need 2 edits in the same function. You can check this backport for the actual diff you need: #3632<https://github.com/OSC/ondemand/pull/3632>

**Comment by johrstrom** [comment]
This was fixed in 3.1.7 and will be fixed in future releases too. Thanks for the ticket!
